lib/elixir/lib/module/types/infer.ex

```diff
@@ -246,6 +246,7 @@ defmodule Module.Types.Infer do
     {:is_port, 1} => {[:port], :boolean},
     {:is_reference, 1} => {[:reference], :boolean},
     {:is_tuple, 1} => {[:tuple], :boolean},
+    {:map_get, 2} => {[{:map, []}, :dynamic], :dynamic},
```

Comment:
Per my comments above, is_map_key should be added too once the fix is pushed. :)